src/docs/asciidoc/hazelcast_overview.adoc

```diff
@@ -293,6 +176,32 @@ In these cases, the partition table in the oldest member is updated with the new
 ownerships. Note that if a lite member joins or leaves a cluster, repartitioning is not triggered
 since lite members do not own any partitions.
 
+[[use-cases]]
+=== Use Cases
+
+Hazelcast can be used:
+
+* to share server configuration/information to see how a cluster performs
+* to cluster highly changing data with event notifications, e.g., user based events, and
+to queue and distribute background tasks
+* as a simple Memcache with Near Cache
+* as a cloud-wide scheduler of certain processes that need to be performed on some members
+* to share information (user information, queues, maps, etc.) on the fly with multiple
+members in different installations under OSGI environments
+* to share thousands of keys in a cluster where there is a web service interface on an
```

Comment:
thousands is now already too low, maybe omit a number as it can be highly dependendant on the size and other factors.